client/client_test.go

```diff
@@ -484,24 +486,68 @@ func concurrentIncrements(db *client.DB, t *testing.T) {
 			// Wait until the other goroutines are running.
 			wgStart.Wait()
 
-			if err := db.Txn(func(txn *client.Txn) error {
-				txn.SetDebugName(fmt.Sprintf("test-%d", i))
-
-				// Retrieve the other key.
-				gr, err := txn.Get(readKey)
-				if err != nil {
-					return err
+			if isMultiphase {
```

Comment:
I originally did that, and saw that I was repeating a lot of logic here. Perhaps we need to do this and add another test?